app/mailers/notifier.rb

```diff
@@ -0,0 +1,10 @@
+class Notifier < ActionMailer::Base
+  def statistic statistic
```

Comment:
@vlasiak Btw, `digest` term could be used here. Think on solid name for this domain and use it other places.